src/engine/Engine.h

```diff
@@ -405,12 +416,14 @@ class Engine : public IEngine, public SignalHandler::Signalable
      */
     String _queryId;
 
+    Map<unsigned, double> _heuristicCost;
 
     /*
       Perform a simplex step: compute the cost function, pick the
       entering and leaving variables and perform a pivot.
+      Return true only if the current assignment is optimal.
```

Comment:
optimal with respect to what?

Reply:
With respect to _heuristicCost. I updated the comment.